tests/whitebox/_misc.py

```diff
@@ -37,6 +37,32 @@
     sys.exit(message)
 
 
+def check_error(obj, expected_err, expected_cause, command_line, expected_code):
+    """
+    Check that the expected exception was raised, and that the cause
+    and exit codes where also as expected, based on the command line arguments
+    passed to the program.
+    :param expected_err: the expected top level exception in the chain
+    :type expecter_err: StratisCliError
+    :param expected_cause: the expected cause of the top level exception
+    :type expected_cause: StratisCliError
```

Comment:
This type is actually more general, go with ```Exception```, although that might be too general.

Reply:
And this is to account for DbusClientError as well, correct? Is it bad practice to be too general? Is there a better practice if you only want to accept DbusClientError and StratisCliError?